### PR TITLE
Reverse stack

### PR DESCRIPTION
**Pull request by sourabhjagtap95**
Issue #1159

### CODE REVIEW

File: code/data_structures/stack/reverse_stack/reverse_stack.c [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+ 
+        /* Insert all the items (held in Function Call Stack)
+           one by one from the bottom to top. Every item is
+           inserted at the bottom */
+        insertAtBottom(top_ref, temp);
+    }
+}
+ 
+/* Driveer program to test above functions */
+int main()
```
Comment (arnavb) [inline code/data_structures/stack/reverse_stack/reverse_stack.c]: Don't just stick `main` in the middle of your code, move it above the functions that are above it, and add function prototypes for them as well.

File: code/data_structures/stack/reverse_stack/reverse_stack.c [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        reverse(top_ref);
+ 
+        /* Insert all the items (held in Function Call Stack)
+           one by one from the bottom to top. Every item is
+           inserted at the bottom */
+        insertAtBottom(top_ref, temp);
+    }
+}
+ 
+/* Driveer program to test above functions */
```
Comment (arnavb) [inline code/data_structures/stack/reverse_stack/reverse_stack.c]: Typo

File: code/data_structures/stack/sort_stack/sort_stack.c [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    (*s) = (*s)->next;
+    free(temp);
+ 
+    return x;
+}
+ 
+// Function to find top item
+int top(struct stack *s)
+{
+    return (s->data);
```
Comment (arnavb) [inline code/data_structures/stack/sort_stack/sort_stack.c]: No  need for parentheses.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**